docs/assistant/security-assistant.asciidoc

```diff
@@ -139,5 +163,19 @@ The fields on this list are among those most likely to provide relevant context
 image::images/add-alert-context.gif[A video that shows an alert being added as context to an AI Assistant chat message]
 +
 When you include a particular event as context, you can use a similar interface to adjust anonymization behavior. Be sure the anonymization behavior meets your specifications before sending a message with the event attached.
-
++
 The *Show anonymized* toggle controls whether you see the obfuscated or plaintext versions of the fields you sent to AI Assistant. It doesn't control what gets obfuscated — that's determined by the anonymization settings. It also doesn't affect how event fields appear _before_ being sent to AI Assistant. Instead, it controls how fields that were already sent and obfuscated appear to you.
+
+* **Knowledge base:** Use retrieval augmented generation to provide specialized knowledge of the Elastic Search Query Language ({esql}) to AI Assistant. For example, with knowledge base active, you can ask AI Assistant to help you write an {esql} query for a particular use-case, or ask it to answer general questions about {esql} syntax and usage. Without knowledge base enabled, AI Assistant will not be able to answer questions about {esql}.
++
+beta::[]
++
+To enable Knowledge base:
++
+- Enable the Elastic Learned Sparse EncodeR (ELSER). This model provides additional context to the third-party LLM. To learn more, refer to https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html#download-deploy-elser[Configure ELSER]
+- Initialize the knowledge base by clicking *Initialize*.
+- Initialize the {ESQL} knowledge base documents by clicking *Initialize*.
+- Move the *Knowledge Base* slider to the `on` position.
```

Comment:
I think this is just a toggle that the user clicks, not a slider that they drags back and forth.
```suggestion
- Turn on the *Knowledge Base* option.
```